components/forms/src/error.rs

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use interrupt_support::Interrupted;
+use sync15_traits::bridged_engine;
+
+#[derive(Debug)]
+pub enum QuotaReason {
+    TotalBytes,
+    ItemBytes,
+    MaxItems,
+}
+
+#[derive(Debug, thiserror::Error)]
+pub enum ErrorKind {
+    #[error("Quota exceeded: {0:?}")]
+    QuotaError(QuotaReason),
```

Comment:
I know this came from `webext_storage`, but forms doesn’t have quotas, so let’s nix these (and `InvalidConnectionType`, `ConnectionAlreadyOpen`, and `WrongApiForClose`, too, I think).

Reply:
yep - I had this comment queued too - remove anything that's not actually used :)